test/extensions/path/uri_template_lib/uri_template_test.cc

```diff
@@ -98,58 +100,53 @@ TEST(ParseRewrite, InvalidRegex) {
-                             {"/{var1}", R"EOF(segments:
-                          - literal: "/"
-                          - capture_index: 1)EOF"},
-                             {"/{var1}/{var1}/{var1}", R"EOF(segments:
-                                        - literal: "/"
-                                        - capture_index: 1
-                                        - literal: "/"
-                                        - capture_index: 1
-                                        - literal: "/"
-                                        - capture_index: 1)EOF"},
-                             {"/{var3}/{var1}/{var2}", R"EOF(segments
-                                        - literal: "/"
-                                        - capture_index: 3
-                                        - literal: "/"
-                                        - capture_index: 1
-                                        - literal: "/"
-                                        - capture_index: 2)EOF"},
-                             {"/{var3}/abc/def/{var2}.suffix", R"EOF(segments:
-                                                - literal: "/"
-                                                - capture_index: 3
-                                                - literal: "/abc/def/"
-                                                - capture_index: 2
-                                                - literal: ".suffix")EOF"},
-                             {"/abc/{var1}/{var2}/def", R"EOF(segments
-                                         - literal: "/abc/"
-                                         - capture_index: 1
-                                         - literal: "/"
-                                         - capture_index: 2
-                                         - literal: "/def")EOF"},
-                             {"/{var1}{var2}", R"EOF(segments
-                                - literal: "/"
-                                - capture_index: 1
-                                - ar_index: 2)EOF"},
-                             {"/{var1}-{var2}/bucket-{var3}.suffix", R"EOF(segments
-                                                      - literal: "/"
-                                                      - capture_index: 1
-                                                      - literal: "-"
-                                                      - capture_index: 2
-                                                      - literal: "/bucket-"
-                                                      - capture_index: 3
-                                                      - literal: ".suffix")EOF"},
-                         })));
+INSTANTIATE_TEST_SUITE_P(
+    ParseRewriteSuccessTestSuite, ParseRewriteSuccess,
+    testing::ValuesIn(
+        std::vector<std::pair<std::string, std::vector<absl::variant<int, std::string>>>>({
+            {"/static",
+             std::vector<absl::variant<int, std::string>>{
```

Comment:
nit: 
absl::variant<int, std::string> => RewriteSegment and
std::vector<absl::variant<int, std::string>> => RewriteSegments.